### PR TITLE
chore: Update dependencies, update test data to match

### PR DESCRIPTION
**Pull request by Moopli**
Signed-off-by: Filip Burlacu <filip.burlacu@securekey.com>

### CODE REVIEW

File: test/bdd/fixtures/demo/.env [hunk 1]
```diff
@@ -60,7 +60,7 @@ FABRIC_BUILDER_FIXTURE_TAG=0.1.2
 
 
 BLOC_DID_METHOD_IMAGE=docker.pkg.github.com/trustbloc/trustbloc-did-method/did-method-rest
-BLOC_DID_METHOD_IMAGE_TAG=0.1.3-snapshot-00c874f
+BLOC_DID_METHOD_IMAGE_TAG=0.1.3-snapshot-57c6171
```
Comment (sudeshrshetty) [inline test/bdd/fixtures/demo/.env:63]: you didn't update VCS_IMAGE_TAG??

File: test/bdd/features/setupfabric.feature [hunk 2]
```diff
@@ -54,9 +54,9 @@ Feature:
...
+    When fabric-cli is executed with args "file upload --url https://peer0-org1.trustbloc.local/.well-known/did-trustbloc --files ./fixtures/discovery-config/sidetree-fabric/config/testnet.trustbloc.local.json;./fixtures/discovery-config/sidetree-fabric/config/peer0-org1.trustbloc.local.json;./fixtures/discovery-config/sidetree-fabric/config/peer0-org2.trustbloc.local.json;fixtures/discovery-config/sidetree-fabric/config/peer0-org3.trustbloc.local.json --idxurl https://peer0-org1.trustbloc.local/file/${fileIdxID} --pwd pwd1 --nextpwd pwd2 --noprompt"
     Then the JSON path "#" of the response has 4 items
-    And the JSON path "0.Name" of the response equals "consortium.json"
+    And the JSON path "0.Name" of the response equals "testnet.trustbloc.local.json"
```
Comment (fqutishat) [inline test/bdd/features/setupfabric.feature:59]: can you change it to peer0-org1.trustbloc.local.json

File: test/bdd/features/setupfabric.feature [hunk 3]
```diff
@@ -65,11 +65,11 @@ Feature:
     And the JSON path "3.Name" of the response equals "peer0-org3.trustbloc.local.json"
     And the JSON path "3.ContentType" of the response equals "application/json"
 
-    Then we wait 10 seconds
+    Then we wait 60 seconds
 
     # Resolve .well-known files
-    When an HTTP request is sent to "https://peer1-org2.trustbloc.local/.well-known/did-trustbloc/consortium.json"
-    Then response from "https://peer1-org2.trustbloc.local/.well-known/did-trustbloc/consortium.json" to client contains value "payload"
+    When an HTTP request is sent to "https://peer1-org2.trustbloc.local/.well-known/did-trustbloc/testnet.trustbloc.local.json"
```
Comment (fqutishat) [inline test/bdd/features/setupfabric.feature:71]: can you change it to peer0-org1.trustbloc.local.json

File: test/bdd/features/setupfabric.feature [hunk 4]
```diff
@@ -65,11 +65,11 @@ Feature:
     And the JSON path "3.Name" of the response equals "peer0-org3.trustbloc.local.json"
     And the JSON path "3.ContentType" of the response equals "application/json"
 
-    Then we wait 10 seconds
+    Then we wait 60 seconds
```
Comment (fqutishat) [inline test/bdd/features/setupfabric.feature:68]: change it to 10

**Comment by DRK3** [comment]
If we merge your PR in, does mine become unneeded? If so, I can close it

**Comment by Moopli** [comment]
@DRK3 
> If we merge your PR in, does mine become unneeded? If so, I can close it

We can merge yours, I just wanted to get ahead of any rebase that might sneak in a version reversion.

**Comment by DRK3** [comment]
> @DRK3
> 
> > If we merge your PR in, does mine become unneeded? If so, I can close it
> 
> We can merge yours, I just wanted to get ahead of any rebase that might sneak in a version reversion.

Sounds good.

**Comment by Moopli** [comment]
The sidetree-fabric demo is currently not working, but sidetree-mock works.

File: test/bdd/features/setupfabric.feature (hunk 3 above)
Comment (Moopli) [reply test/bdd/features/setupfabric.feature:71]: This won't resolve the problem (as we discussed)

**Pull request merged**